openpype/hosts/fusion/plugins/publish/collect_renders.py

```diff
@@ -0,0 +1,30 @@
+import pyblish.api
+from pprint import pformat
+
+
+class CollectFusionRenders(pyblish.api.InstancePlugin):
+    """Collect current saver node's render Mode
+
+    Options:
+        local (Render locally)
+        frames (Use existing frames)
+
+    """
+
+    order = pyblish.api.CollectorOrder + 0.4
+    label = "Collect Renders"
+    hosts = ["fusion"]
+    families = ["render"]
+
+    def process(self, instance):
+        self.log.debug(pformat(instance.data))
```

Comment:
Please do not log the full instance - it's very slow as data tends to grow large.
```suggestion
```